R/package_spec.R

```diff
@@ -5,17 +5,22 @@
 #'
 #' @param name name of the package.
 #' @param repos repository where package with given name should identified.
+#' @param version package_version object specifying minimal version required
+#'  by packages depending on this package.
+#' @param op operator used with \code{version}.
 #' @param path path to the source of the package (either bundled or not). URLs
 #' are acceptable.
 #' @param ... parameters passed to downstream constructors
 #'
 #' @family specs
 #' @export
-package_spec <- function(name = NULL, repos = NULL) {
+package_spec <- function(name = NULL, repos = NULL, version = NULL, op = NULL) {
```

Comment:
```suggestion
package_spec <- function(name = NULL, repos = NULL, version = package_version("0.0"), op = ">=") {
```

I think we should give some defaults here. If not, when you set default values it's possible that `op == NULL` and version is provided, leading to imputing with `op == ">"` and disallowing the version number specified.

Reply:
Good point, thanks!